swap_meet/clothing.py

```diff
@@ -1,2 +1,16 @@
-class Clothing:
-    pass
\ No newline at end of file
+import uuid
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+    def __init__(self, id=uuid.uuid4().int, fabric="Unknown", condition=0, age=0):
```

Comment:
We should not be setting id using the uuid function in the constructor. This makes it so that every single Clothing instance has the same id. Instead, you should do what you did in the Item class and set the default to None and then let the parent class take care of creating a new id for you. This happens to pass our tests, but try creating two Clothing instances and you'll see both have the same id.

This applies to Decor and Electronics as well.